game/match.py

```diff
@@ -3,16 +3,17 @@
 from game.player import Player
 from itertools import cycle
 
-class GameIsOver(Exception): # Where should this exception be placed?
+class GameIsOver(Exception):
     def __init___(self, winner):
         self.winner = winner
 
 class Match(object):
-    def __init__(self, player_one = Player('red'),
-                 player_two = Player('blue'), board = GameBoard()):
+    def __init__(self, player_one, player_two, board):
         self._board = board
         self._active_player = None
-        self._players = [player_one, player_two]
+        self._player_one = player_one
+        self._player_two = player_two
+        self._players = [self._player_one, self._player_two]
```

Comment:
yup, the code would look like:

```
def players(self):
    return [self._player_one, self._player_two]
```

and in your code, instead of doing `self._players` do `self.players()`